src/pom.xml

```diff
@@ -135,8 +135,9 @@
         <sparkbench.mr.version>2</sparkbench.mr.version>
       </properties>
       <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
My concern is that when we specify "-P MR1", it seems profile MR2 is still active because it is active by default. It becomes "-P MR1 -P MR2" actually unless we use "-P !MR2" to disable it?